modules/openapi-generator/src/main/resources/dart2/serialization/native/native_enum.mustache

```diff
@@ -4,12 +4,12 @@ class {{{classname}}} {
   const {{{classname}}}._(this.value);
 
   /// The underlying value of this enum member.
-  final {{{dataType}}} value;
+  final {{{dataType}}}? value;
```

Comment:
can this nullability depend on whether `enumVars` contains `null`?

Reply:
do you mean "if any enumVar is null then this is nullable`?

Reply:
yeah thats what I mean

I'm guessing `if any enumVar is null` would need to be exposed on the java side?